src/main/kotlin/com/devfloor/untitled/comment/domain/Comment.kt

```diff
@@ -51,4 +51,7 @@ class Comment(
     @Column(name = "content")
     var content: String = content
         protected set
+
+    fun modify(comment: Comment) = comment.content
```

Comment:
여기 이렇게 되어야하는 것 아닌가요 !?
```suggestion
    fun modify(comment: Comment) {
        this.content = comment.content
    }
```

Reply:
제가 원래 이렇게 구현했었는데
형님이 첫 리뷰해주신 부분에 블록바디 말고 표현식으로 쓰라고하셔서 근데 저상황에서 저렇게쓰는게 맞는건지 모르겠는데 변경했던거거든용.. ㅜ

Reply:
아 그렇네요 ! 해당 리뷰 내용은 one line인 경우에 블록바디 말고 expression으로 하라는 컨벤션 이야기를 한 것이었는데 생각해보니 저렇게 할당해야 하는 곳에는 expression이 불가능하니 이 경우에 블록바디로 쓰는게 맞을것같네요 !

참고로 Assignments are not expressions 라고 합니다 !

Reply:
그쳐? 도저히모르겠는데 저렇게 써보니까 에러안뜨길래..ㅋㅋㅋㅋㅋ 넵 감사합니당